src/logOut.js

```diff
@@ -1,6 +1,23 @@
+let timerLogOut = () => {
+    let timeLeft = 5;
+
+    let timer = setInterval(function() {
+        if (timeLeft == 5) console.log(`${timeLeft} секунд,`);
+        if (timeLeft > 1 && timeLeft < 5) console.log(`${timeLeft} секунды,`);
+        if (timeLeft == 1) console.log(`${timeLeft} секунда`);
```

Comment:
```suggestion
        if (timeLeft === 1) console.log(`${timeLeft} секунда`);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Это очень важно знать!